UnityProject/Assets/Mirror/Runtime/NetworkServer.cs

```diff
@@ -1522,47 +1533,48 @@ static void BroadcastToConnection(NetworkConnectionToClient connection)
-                    //
-                    // NOTE: this is what we did before push->pull
-                    //       broadcasting. let's keep doing this for
-                    //       feature parity to not break anyone's project.
-                    //       TODO make this more simple / unnecessary later.#
-                    //CUSTOM UNITYSTATION CODE// commented this out \/
-                    //identity.ClearDirtyComponentsDirtyBits();
-                }
+                //CUSTOM UNITYSTATION CODE// == null = lag
+                // if (identity != null)
+                // {
+                //CUSTOM UNITYSTATION CODE//
+	            if (identity.isDirty == false)
+	            {
+		            continue;
+	            }
+	            ////////////////////////////
+	            // get serialization for this entity viewed by this connection
+	            // (if anything was serialized this time)
+	            NetworkWriter serialization = GetEntitySerializationForConnection(identity, connection, out bool isNull);
+	            if (isNull == false)
+	            {
+		            EntityStateMessage message = new EntityStateMessage
+		            {
+			            netId = identity.netId,
+			            payload = serialization.ToArraySegment()
+		            };
+		            connection.Send(message);
+	            }
+
+	            // clear dirty bits only for the components that we serialized
+	            // DO NOT clean ALL component's dirty bits, because
+	            // components can have different syncIntervals and we don't
+	            // want to reset dirty bits for the ones that were not
+	            // synced yet.
+	            // (we serialized only the IsDirty() components, or all of
+	            //  them if initialState. clearing the dirty ones is enough.)
+	            //
+	            // NOTE: this is what we did before push->pull
+	            //       broadcasting. let's keep doing this for
+	            //       feature parity to not break anyone's project.
+	            //       TODO make this more simple / unnecessary later.#
+	            //CUSTOM UNITYSTATION CODE// commented this out \/
+	            //identity.ClearDirtyComponentsDirtyBits();
+                //}
                 // spawned list should have no null entries because we
                 // always call Remove in OnObjectDestroy everywhere.
                 // if it does have null then someone used
                 // GameObject.Destroy instead of NetworkServer.Destroy.
-                else Debug.LogWarning("Found 'null' entry in observing list for connectionId=" + connection.connectionId + ". Please call NetworkServer.Destroy to destroy networked objects. Don't use GameObject.Destroy.");
+                //else Debug.LogWarning("Found 'null' entry in observing list for connectionId=" + connection.connectionId + ". Please call NetworkServer.Destroy to destroy networked objects. Don't use GameObject.Destroy.");
```

Comment:
Why comment this?

Reply:
it's the other side of the null check that I removed

Reply:
Well yeah but checking the IsNull variable conceptually does the same and if it is true then the log is relevant, I think

Reply:
It's only elseon the null date Needing to be sent